src/styles/main.css

```diff
@@ -0,0 +1,98 @@
+* {
+  margin: 0;
+  padding: 0;
+  box-sizing: border-box;
+}
+
+body {
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  height: 100vh;
+}
+
+.calendar {
+  counter-reset: day-number 0;
+  display: flex;
+  flex-wrap: wrap;
+  width: calc(100px * 7 + 1px * 6 + 10px * 2);
+  padding: 10px;
+  gap: 1px;
+}
+
+.calendar__day {
+  width: 100px;
+  height: 100px;
+  background-color: #eee;
+  border: 1px solid #000;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  position: relative;
+}
+
+.calendar__day::before {
+  content: counter(day-number);
+  counter-increment: day-number;
+  font-size: 30px;
+  font-family: Arial, sans-serif;
+  color: #000;
+  position: absolute;
+}
```

Comment:
Instead of using counter, you can simply use a loop in SCSS to generate the necessary classes with the corresponding content.